idioms/angularNestJsLambda/backend/models/entity/entity.ts.model

```diff
@@ -0,0 +1,8 @@
+import {Column, DataType, Model, Table} from 'sequelize-typescript';
+
+@Table
+export class {{Entity singular name}} extends Model<{{Entity singular name}}> {
+
+    {{Entity attributes}}
```

Comment:
` {{For each Entity attributes: ./entity.model.attribute.*.idiom}}`
This wildcard should be used to several options of idioms in this place.